src/main/test/java/seedu/clover/ParserTest.java

```diff
@@ -0,0 +1,13 @@
+import clover.util.Parser;
+import clover.util.Parser.Commands;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
You should explicitly list it out as:

`import static org.junit.jupiter.api.Assertions.assertEquals;`